stripe-core/detekt-baseline.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Generates a detekt baseline to start from.

Reply:
@jameswoo-stripe is it possible to configure detekt so that unintentional calls to connection is caught?